frontend/pages/admin.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+definePageMeta({
+  middleware: 'admin'
+})
+</script>
+
+<template>
+  <div>
+    <NuxtPage />
+  </div>
```

Comment:
I think you can drop the wrapping `div` here. If not, you should be able to use `template` instead

Reply:
SG